docs/content/file-formats.md

```diff
@@ -30,6 +30,44 @@ In addition to the normal fields available on a [Porter Installation document](/
 
 [Installation]: /operator/glossary/#installation
 
+## CredentialSet
+
+See the glossary for more information about the [CredentialSet] resource.
+
+```yaml
+apiVersion: porter.sh/v1
+kind: CredentialSet
+metadata:
+  name: credentialset-sample
+spec:
+  schemaVersion: 1.0.1
+  namespace: operator
+  name: porter-test-me
+  credentials:
+    - name: test-credential
+      source:
+        secret: test-secret
+```
+
+| Field        | Required | Default                            | Description                                                 |
+|--------------|----------|------------------------------------|-------------------------------------------------------------|
+| credentials  | true     |                                    | List of credential sources for the set |
+| agentConfig  | false    | See [Agent Config](#agentconfig)   | Reference to an AgentConfig resource in the same namespace. |
+| porterConfig | false    | See [Porter Config](#porterconfig) | Reference to a PorterConfig resource in the same namespace. |
+
+### Credential Source
```

Comment:
Can you merge this table into the Credential Set section? This page should document each individual kubernetes manifest that you can use with the Porter operator, and Credential Source isn't a standalone resource.

Here's an example of how to document nested types from the Porter Credential Set docs.

https://release-v1.porter.sh/reference/file-formats/#credential-set